crates/runtime/tests/vmcalls_tests.rs

```diff
@@ -247,7 +247,7 @@ fn vmcalls_load160() {
     let view = &memory.view::<u8>()[ptr as usize..(ptr as usize + 20)];
     let bytes: Vec<u8> = view.iter().map(|cell| cell.get()).collect();
 
-    assert_eq!(target_addr, Address::from(&bytes[..]));
+    assert_eq!(target_addr, Address::new(&bytes[..]));
```

Comment:
Why not keep the previous `from` ?

Reply:
We can't implement [`std::convert::From` for `BytesPrimitive` implementors, we can only provide `BytesPrimitive::foobar` methods. A rename seemed like a small change in exchange for a number of benefits.